llama-index-core/llama_index/core/callbacks/token_counting.py

```diff
@@ -23,21 +36,65 @@ def __post_init__(self) -> None:
         self.total_token_count = self.prompt_token_count + self.completion_token_count
 
 
+def get_tokens_from_response(
+    response: Union["CompletionResponse", "ChatResponse"]
+) -> Tuple[int, int]:
+    """Get the token counts from a raw response."""
+    usage = response.raw.get("usage", {})
```

Comment:
This will error out saying the following.
 'ChatCompletion' object has no attribute 'get'
 
![image](https://github.com/user-attachments/assets/bfbf09a2-a4c2-4d18-8474-9cd41dfa56da)

We will have to access the usage attribute like
response.raw.usage
![image](https://github.com/user-attachments/assets/4b497d3b-48f8-4f59-aa59-e0120430bc08)

Reply:
We can just model dump the raw response if its not already a dict